fastlane/swift/Fastlane.swift

```diff
@@ -697,6 +697,7 @@ public func apteligent(dsym: String? = nil,
    - endpoint: Artifactory endpoint
    - username: Artifactory username
    - password: Artifactory password
+   - apiKey: Artifactory API key
```

Comment:
Code in all `.swift` files are auto-generated during the release process - could you revert these changes, please? 🙏